ChangeLog

```diff
@@ -1,5 +1,139 @@
+
+- Build: Enable signing of macOS binary with self signed certificate. If a code signing certificate is set up but no notarization information given, Autobuild will now sign the builds assuming a self signed certificate is given (#2944).
+  (contributed by @ann0see)
+
+- Internal: Fix C++ build warnings regarding copy constructor and assignment functions (#2946).
+  (contributed by @hoffie)
+
+- Internal: Reduced permissions in Autobuild for security hardening (#2953).
+  (contributed by @ann0see)
+
+- Build: Updated bundled JACK (Windows-only) to version 1.9.21.20221111 (#2960).
+  (contributed by @app/github-actions)
+
+- Build: Updated bundled Qt6 to version 6.4.1 (#2961).
+  (contributed by @app/github-actions)
+
+- Build: Updated bundled Qt6 to version 6.4.2 (#2991).
+  (contributed by @app/github-actions)
+
+- Make checkboxes self-labelling (#2996).
+  (contributed by @pljones)
+
+- Translations updated from Hosted Weblate (#2998).
+  (contributed by @weblate)
+
+- Internal: The JACK build will now install directly from the JACK GitHub repository (#3003).
+  (contributed by @ann0see)
+
+- CONDENSE with https://github.com/jamulussoftware/jamulus/pull/2833 (#3008).
+  (contributed by @ann0see)
+
+- Server: Use native file selection dialog for recording directory and persistence file to allow read and write access outside of the macOS sandbox (#3009).
+  (contributed by @ann0see)
+
+- Internal: Improved Qt Creator 5 compliance of Jamulus.pro (#3010).
+  (contributed by @pljones)
+
+- Linux: Debian users can now use the official PPA to get automatic updates. See the Linux install page on jamulus.io for more information (#3013).
+  (contributed by @ann0see)
+
+- Internal: The JACK build will now install directly from the JACK GitHub repository (#3014).
+  (contributed by @ann0see)
+
+- Rename `--directoryserver` to `--directoryaddress` (and internal changes) (#3017).
+  (contributed by @pljones)
+
+- Updated clang version to Ubuntu LTS version (#3020).
+  (contributed by @pljones)
+
+- Build: Bump dev-drprasad/delete-tag-and-release from 085c6969f18bad0de1b9f3fe6692a3cd01f64fe5 to 0885ecc4123e071dd1a8be04d3a0578f5390581c (#3027).
+  (contributed by @app/dependabot)
+
+- Build: Updated bundled JACK (Windows-only) to version 1.9.22 (#3031).
+  (contributed by @ann0see)
+
+- Internal: Improved maintainability of Jamulus.pro by using global definitions (#3046).
+  (contributed by @ann0see)
+
+- rename `directoryServer` to `directory` in `jamulusserver/getServerProfile` response (#3048).
+  (contributed by @pljones)
+
+- Modernise / fix some Python and added linting to CI (#3052).
+  (contributed by @declension)
+
+- Save the HTML formatting in Server Welcome Message. (Applies to the GUI Server.) (#3058).
+  (contributed by @pljones)
+
+- Mac: Upgrade legacy build to a new but unsupported SDK. This hopefully still allows the legacy build to exist, but no guarantees are given. You are advised to upgrade your OS (#3066).
```

Comment:
I think this needs to be shorter and clearer.  Something like "Change minimum support macOS version to X.Y.  Upgrade your OS if you find Jamulus isn't working."

Reply:
Or something like:
> - Build: Windows 64bit releases use Qt 6.4.3 now. Jamulus will from now on only support Windows 10 and above (#2300).
>  (contributed by @hoffie)

Reply:
Should be better now.